conf/config.properties

```diff
@@ -255,6 +255,10 @@ classification.language.spanish=que,de,no,a,la,el,es,y,en,lo,un,por,qu
 classification.language.dutch=de,van,een,het,en,in,is,dat,op,te,De,zijn,voor,met,die,niet,aan,er,om,Het,ook,als,dan,maar,bij,of,uit,nog,worden,door,naar,heeft,tot,ze,wordt,over,hij,In,meer,jaar,was,ik,kan,je,zich,al,hebben,geen,hun,we,wat,Een,Maar,werd,moet,wel,kunnen,Dat,nu,dit,deze,zal,Ik,veel,zo,En,andere,nieuwe,zou,twee,moeten,onder,eerste,haar,Van,wil,tegen,men,mensen,gaat,tussen,grote,waar,goed,maken,dus,alleen,Hij,Op,frank,ons,u,daar,na,had,gaan,alle,Als,Er,��n
 
 # flags
+flag.log.dao = true;
+flag.severe.dao = true;
+flag.debug.dao = true;
+flag.trace.dao = true;
```

Comment:
> And these log levels (trace, info, warn, debug) dealt here are logged as `INFO` in the loklak.log

This is not how it works. The log level specifies about how much important logs should be kept on a particular medium. If it is `INFO`, then all the messages having equal or more priority are logged. Similarly, when it is `WARN`, no `INFO` type logs will be displayed, and so on.

Also, having to choose for the types of messages to be logged is not logical. Why would anyone wish to see `INFO` but ignore `WARN` type messages?